docs/man/rpm-config.5.scd

```diff
@@ -0,0 +1,217 @@
+RPM-CONFIG(5)
+
+# NAME
+*rpm-config* - rpm runtime macro configuration
+
+# SYNOPSIS
+_NAME_ _VALUE_
+
+# DESCRIPTION
+The primary configuration mechanism in *rpm* is via macros. On startup,
+*rpm* reads a set of macro files (*rpm-macrofile*(5)) as defined
+in the macro path.
+
+Each file or *glob*(7) pattern in the colon-separated macro path is
+read sequentially by *rpm* for macro definitions. *%{\_target}* is
+expanded to the detected <arch>-<os> platform. Tildes are expanded
+to the value of the environment variable *HOME*.
+
+If a macro is defined multiple times, the last entry wins. The default
```

Comment:
It's a bit annoying that when reading this paragraph, you have no clue what those files are and where they reside. So maybe add a "see FILES for the paths" or similar here? (Same for the other man page)

Reply:
That, or put the files (back to) SYNOPSIS maybe. There is a certain point to having the paths up-front there in SYNOPSIS, even if it seems wrong otherwise.

Reply:
Indeed. In my experience, I tend to look for the *filenames* first when opening up a man page about configuration :smile: So +1 on putting them back to SYNOPSIS.

Reply:
Another idea could be putting FILES before DESCRIPTION in section 5 manuals :thinking:

Reply:
Pushed this (FILES at top) as a fixup for easy reversal. It looks strange but so does files in SYNOPSIS. But, I think everybody agrees having the file information right there is very handy :smile:

Reply:
Oh, yep, that sounds like a good idea, let's try.